src/galaxy/CustomSystem.cpp

```diff
@@ -94,6 +94,14 @@ static int l_csb_new(lua_State *L)
 		lua_settop(L, 1); return 1;                        \
 	}
 
+#define CSB_FIELD_SETTER_STRING(luaname, fieldname)    \
+	static int l_csb_ ## luaname (lua_State *L) {        \
+		CustomSystemBody *csb = l_csb_check(L, 1);         \
+		std::string value = luaL_checkstring(L, 2);				 \
+		csb->fieldname = value;                            \
+		lua_settop(L, 1); return 1;                        \
+	}
```

Comment:
Nitpick, compared to the other `#defines` their line breaking `\` are straight, yours are "out of line".